src/storage/src/source/postgres/replication.rs

```diff
@@ -415,7 +427,7 @@ pub(crate) fn render<G: Scope<Timestamp = MzOffset>>(
 /// keepalive messages with the provided `uppers` stream.
 ///
 /// The returned stream will contain all transactions that whose commit LSN is beyond `resume_lsn`.
-async fn raw_stream<'a>(
+async fn raw_stream<'a, G>(
```

Comment:
This function does not need to be generic over a scope. Just replace all `G::Timestamp` below with `MzOffset`

Reply:
good point!